spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> P.S. `raw` is already present in nodes generated by all the existing parsers, so it's not breaking backwards compatibility but standardizing it (as we did with most "non-standard" features when ESTree was initiated).

Again I say, parsers are not the only source of ASTs. Do you really want to make `raw` required, even if only for directives?

> we chould solve CST in general

Agreed, but my impression was that full CST components will be optional, so they can't be relied upon for directive testing.

> have some parenthesized flag on Expression iself

It's true, AFAICT `directive` is not required if we have both `Expression#parenthesized` and `Literal#raw`.